go-controller/pkg/factory/factory.go

```diff
@@ -480,6 +520,31 @@ func NewWatchFactory(c kubernetes.Interface) (*WatchFactory, error) {
 	return wf, nil
 }
 
+func (wf *WatchFactory) EgressFirewallWatchFactory() error {
```

Comment:
consider: naming this `InitializeEgressFirewallWatchFactory` or something to that effect since methods (most of the times) are better readable as verbs.

Reply:
+1

Reply:
done